.github/workflows/docker-publish.yml

```diff
@@ -13,15 +13,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Export REACT_APP_CE_CURRENT_VERSION env var
+      - name: Export REACT_APP_APP_VERSION env var
         run: |
           git fetch --prune --unshallow
-          echo "REACT_APP_CE_CURRENT_VERSION=$(git describe --tags --abbrev=0) ($(git rev-parse --abbrev-ref HEAD):$(git log -1 --format=%h))" >> $GITHUB_ENV
+          echo "REACT_APP_APP_VERSION=$(git describe --tags --abbrev=0) ($(git rev-parse --abbrev-ref HEAD):$(git log -1 --format=%h))" >> $GITHUB_ENV
+      - name: Build npm package
+        run: |
+          npm ci
+          npm run build
       - name: Publish to Registry
         uses: docker/build-push-action@v1
         with:
           username: ${{ secrets.pcicdevops_at_dockerhub_username }}
           password: ${{ secrets.pcicdevops_at_dockerhub_password }}
+          dockerfile: docker/Dockerfile
           repository: pcic/climate-explorer-frontend
```

Comment:
Are you concerned about the deprecation warning on this?
https://github.com/pacificclimate/climate-explorer-frontend/pull/471/checks#step:6:1

And maybe that would address [this warning](https://github.com/pacificclimate/climate-explorer-frontend/pull/471/checks#step:6:20).

Reply:
Aye, I am concerned by it in so far as I do think we should fix it, but as it predates this issue so I haven't looked into fixing it just yet. I'll roll out these changes to each app (to fix startup times) then I'll go back and review updating that workflow to the latest version.